crates/ruff/src/rules/flake8_use_pathlib/rules/os_path_getatime.rs

```diff
@@ -0,0 +1,42 @@
+use ruff_diagnostics::Violation;
+use ruff_macros::{derive_message_formats, violation};
+
+/// PTH203
```

Comment:
I think i'd rather place this (and the other two respectively) at [Ref1]. Currently, it's a doc comment (`///` vs `//`), so it would be the first line of the documentation ahead of "#What it does"